src/components/Timeline/Views/Wave/Wave.tsx

```diff
@@ -17,10 +17,10 @@ import { useMemo } from "react";
 import { WaveSurferParams } from "wavesurfer.js/types/params";
 
 const ZOOM_X = {
-  min: 10,
+  min: 300,
   max: 1500,
   step: 10,
-  default: 10,
+  default: 300,
```

Comment:
Seeing as these values are spread on the `Range`, would it not be better to change `default` to `defaultValue` and just use that instead? Seems more semantic in the usage, unless `default` is also operating as its own value within the component. I just find it unnerving when using keywords as attributes.

Reply:
Ah, I see what you mean. I think we need both though.